stonesoup/platform/tests/test_platform_simple.py

```diff
@@ -146,29 +146,29 @@ def radars_2d():
     # Create 5 simple radar sensor objects
     radar1 = RadarRangeBearing(
         ndim_state=4,
-        mapping=measurement_mapping,
+        pos_mapping=measurement_mapping,
```

Comment:
I've changed all platforms and sensors to use position_mapping and velocity_mappings, i have also changed the measurement models to use velocity_mapping for the range rate cases. I've not changed mapping which is used by all the models as there is no reason that it will uniquely map to position for all measurement models.